src/btouch.cs

```diff
@@ -103,7 +103,7 @@ string GetAttributeLibraryPath ()
 			} else if (target_framework == TargetFramework.Xamarin_Mac_2_0_Mobile) {
 				return Path.Combine (GetSDKRoot (), "lib", "bgen", "Xamarin.Mac-mobile.BindingAttributes.dll");
 			} else {
-				throw ErrorHelper.CreateError (1043, "Internal error: unknown target framework '{0}'.", target_framework);
+				throw ErrorHelper.CreateError (1053, target_framework);
```

Comment:
why has the number changed ?
`git grep 1053 src` returns me nothing

Reply:
According to `generator-errors.md ` the error message matches 1053, not 1043. I'm assuming whoever put the error code as 1043 in btouch.cs made a mistake.
<img width="1992" alt="Screen Shot 2019-12-04 at 1 49 14 PM" src="https://user-images.githubusercontent.com/54864665/70171600-0652fd00-169d-11ea-8e2f-3f0d7974684c.png">